src/test/java/edu/unc/lib/boxc/migration/cdm/services/SipServiceTest.java

```diff
@@ -1630,4 +1633,14 @@ private String convertResourcePropertiesToString(Resource resource) {
                 .map(Statement::toString)
                 .collect(Collectors.joining("\n"));
     }
+
+    private String mappingBody(String... rows) {
```

Comment:
Since the SipServiceTest has a lot of mappings files going on, I'd suggest giving this a scoped name like `altTextMappingBody`